src/libs/Navigation/types.ts

```diff
@@ -79,6 +79,12 @@ type CentralPaneScreensParamList = {
     [SCREENS.SETTINGS.SUBSCRIPTION.ROOT]: undefined;
 };
 
+type SearchNavigatorParamList = {
+    [SCREENS.SEARCH.BOTTOM_TAB]: undefined;
+    [SCREENS.SEARCH.CENTRAL_PANE]: undefined;
+    [SCREENS.SEARCH.REPORT_RHP]: undefined;
+};
+
```

Comment:
Hey @Zakpak0, I am working on some navigation changes and I noticed these new types. Could you please explain to me why this type is necessary? I don't see any navigator named `SearchNavigator`.

Reply:
Np @adamgrzybowski , happy to.
If I recall correctly, I was having a hard time finding an existing type that defined the search tab's routes in isolation.
I created that to supplement the creation of this type [here](https://github.com/Expensify/App/pull/44931/files/706148d46850cdd2cbcbe8201ef32f91eebbbf04#diff-5b0e04a21dcf344490f93813e7d9c49413bbf31dba01fcd2df358cd334970806R1119-R1120).
Maybe a better name for it would be **SearchTabRoutes**.
This is my first time touching the code base and I was attempting to follow the patterns I saw.
Even if they were a bit naive.

Reply:
> I am working on some navigation changes

Please feel free to mini-refactor if you're working on this at the moment and you see fit, as long as this PRs changes won't throw type errors I think you're good to go!